### DISCUSSION TITLE
How to open learning materials?

### DISCUSSION THREAD

**Discussion by NaveenkumarMD** [microsoft/Microsoft-Fabric-workload-development-sample#170] [Help]
I am using window.open to take the user to the documentation page but that doesn't seem to work?
![image](https://github.com/user-attachments/assets/03b1bd8f-399a-4378-bdaf-d8d8adca5a3b)
```json
  "homePage": {
    "learningMaterials": [
      {
        "title": "Learning Material",
        "description": "SQLserver to fabric documentation",
        "introduction": "SQLserver to fabric documentation",
        "onClick": {
          "action": "open.learningMaterial.dialog1"
        },
        "image": "assets/striimLogo.png"
      }
    ],
```

index.worker.ts
```javascript
      case "open.learningMaterial.dialog1":
        window.open("www.google.com");
        return;
```

**Comment by q-benwillis** [comment]
Rather than using `onClick` you can just use `link`, see: https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/discussions/27#discussioncomment-11151197

**Comment by NaveenkumarMD** [comment]
Do I need to update any packages after changing it to link?
Because it doesn't work

**Reply by yonadiv** [reply]
I am not sure which packages you refer to and in what scenario you are using, but after you do a manifest change as described you will need to build and restart the boilerplate FE if in devmode, or update the packages and reupload the package if in test mode (package upload and adding the new uploaded package to the tenant through the admin portal)
